Fill.py

```diff
@@ -136,33 +136,92 @@ def fill_restrictive(world: MultiWorld, base_state: CollectionState, locations:
     itempool.extend(unplaced_items)
 
 
+def fast_fill(world: MultiWorld, locations: typing.List[Location],
+                     itempool: typing.List[Item], single_player_placement: bool = False) -> None:
+    unplaced_items: typing.List[Item] = []
+    placements: typing.List[Location] = []
+    swap = 0
```

Comment:
swap gets set to 0 and then incremented later, but nothing is ever done with this value.